server/models.js

```diff
@@ -0,0 +1,31 @@
+const db = require('../database/connect');
+
+module.exports = {
```

Comment:
Are you defining your functions directly in the module.exports object? I'm not sure if it's a problem or not but I've never seen it done like that.

Reply:
@matt-leyrat, I think I've seen it in some of our sprints!

Reply:
Right on, don't fix it if it ain't broke!